privaterelay/plans.py

```diff
@@ -557,15 +557,18 @@ def _cached_country_language_mapping(
 
     mapping: PlanCountryLangMapping = {}
     for relay_country in relay_maps.get("by_country", []):
-        assert relay_country not in mapping
+        if relay_country in mapping:
+            raise ValueError("relay_country should not be in mapping.")
```

Comment:
Nit: these will be harder to debug, since we won't know _which_ `relay_county` was already in the mapping. However, we can expand the error message when we get to that point, instead of making every `ValueError` include the value.

Reply:
This was the change that made me wonder: should I go back thru this whole PR and include the value in every `ValueError` message while I'm in here? I don't mind doing that now rather than later.